modules/ROOT/pages/bonita-bpm-overview.adoc

```diff
@@ -25,8 +36,24 @@ Users can use Bonita Portal (_User_ profile) to view and complete the process ta
 
 [#platform]
 
+[.card-section]
 == Bonita Runtime
 
+[.card.card-index]
+--
+xref:engine.adoc[[.card-title]#Bonita Engine# [.card-body.card-content-overflow]#pass:q[After reading what follows, you may want to know more]#]
```

Comment:
Same suggestion, don't write "Read what it's after" :)

```suggestion
xref:engine.adoc[[.card-title]#Bonita Engine# [.card-body.card-content-overflow]#pass:q[Execute your business process automation applications in the Bonita Engine]#]
```

Reply:
changed into something different